internal/dkg/execution.go

```diff
@@ -136,17 +137,17 @@ func (d *Process) executeAndFinishDKG(ctx context.Context, beaconID string, conf
 			return err
 		}
 
+		err = d.store.SaveFinished(beaconID, finalState)
```

Comment:
this was a suggestion from Nicolas in the dkg PR, so we don't lose state if the beaconprocess fucks up